docs/5.0/preview/database-access.md

```diff
@@ -44,23 +44,82 @@ The following diagram shows an example Database Access setup:
 
 ## Schedule
 
-!!! info
-    Teleport Database Access is under active development.
-    The alpha release will include support for PostgreSQL, including Amazon RDS and Aurora.
+Teleport Database Access is under active development. The alpha release will
+include support for PostgreSQL, including Amazon RDS and Aurora.
 
-[Database Access Release Schedule](./upcoming-releases.md#release-schedule)
+See [release schedule](./upcoming-releases.md#release-schedule).
 
-## Setup
+## Configure PostgreSQL
 
-Let's setup a sample Teleport Database Access deployment.
+### On-Prem PostgreSQL
 
-We will create a Teleport cluster and connect a PostgreSQL-flavored AWS Aurora
-database to it via Teleport database proxy.
+!!! note
+    This section explains how to configure an on-premise instance of PostgreSQL
+    to work with Teleport Database Access. For information about configuring
+    AWS RDS/Aurora see the [section below](#aws-rdsaurora-postgresql).
 
-### Setup Aurora
+#### Create Certificate/Key Pair
 
-Teleport Database Access uses IAM authentication with AWS RDS and Aurora
-databases which can be enabled with the following steps.
+Teleport uses mutual TLS authentication so PostgreSQL server must be configured
+with Teleport's certificate authority and certificate/key pair that Teleport
+can validate.
+
+To create these secrets, use `tctl auth sign` command. Note that it requires a
+running Teleport cluster and [should be run](https://goteleport.com/teleport/docs/architecture/overview/#tctl)
+on the auth server.
+
+```sh
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
```

Comment:
Added a comment for lazy people not reading below. Do we have any thoughts / recommendations for best practices for shorter lived certs / automation for getting new certs. 

```suggestion
# Export Teleport's certificate authority for host db.example.com with a one year validity period. 
$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
```

Reply:
It would be good to expand on our best practices for certificate automation. 

Does Postgres have any best practices for dealing with rotating certs? e.g. Does `postgresql.conf` cache these credentials / What's the process for updating certs on production systems.

Reply:
I don't think Postgres supports any sort of automation for certificates rotation, so it will likely require database restart.

For now, I've just added a note that it has to be considered when rotating host CA, we might need to expand on this topic more in one of future iterations.